config/manifests/bases/params.env

```diff
@@ -0,0 +1 @@
+odh-codeflare-operator-controller-image=quay.io/opendatahub/codeflare-operator:v1.0.0
```

Comment:
@astefanutti Do you know who/when/how builds `quay.io/opendatahub/codeflare-operator`? Should we build it?

Reply:
@sutaakar ideally it should be build from the ODH/CFO fork, by the same same workflow with have here in PCF/CFO, just changing the image org?

Reply:
We have a workflow for building this image in odh/cfo [here](https://github.com/opendatahub-io/codeflare-operator/blob/main/.github/workflows/build-and-push.yaml)

Reply:
Right, I just overlook it. Thanks.

Reply:
In general the flow is not ideal as CFO image here is referenced sooner than it is actually built.